server/main.py

```diff
@@ -1,8 +1,24 @@
+from flask import Flask, request, jsonify
+import openpyxl
 from flask_restful import Api
 from routes.student import StudentAPI, StudentsAPI
+import pandas as pd
 
+app = Flask(__name__)
```

Comment:
This is already called in the main.py

Reply:
Or probably you can move the code to main.py for now, because Flask() is called there